clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreBufferService.java

```diff
@@ -211,6 +212,18 @@ private static void processRecord(
     }
   }
 
+  private static void processCommand(
+      CommandQueueNode cmd,
+      StoreIngestionTask ingestionTask,
+      PartitionConsumptionState pcs) {
+    // We only support SYNC_OFFSET command for now.
+    if (cmd.getCommandType() != CommandQueueNode.CommandType.SYNC_OFFSET) {
+      throw new VeniceException("Unsupported command type: " + cmd.getCommandType());
+    }
+
+    cmd.executeSync(() -> ingestionTask.updateOffsetMetadataAndSyncOffset(pcs));
```

Comment:
Nit,
shall we log an `INFO` message here?

Reply:
Sure, will do.